lib/pasarguard-backup.sh

```
@@ -0,0 +1,1314 @@
+remove_backup_service() {
+    colorized_echo red "in process..."
+
+    sed -i '/^# Backup service configuration/d' "$ENV_FILE"
+    sed -i '/BACKUP_SERVICE_ENABLED/d' "$ENV_FILE"
+    sed -i '/BACKUP_TELEGRAM_BOT_KEY/d' "$ENV_FILE"
+    sed -i '/BACKUP_TELEGRAM_CHAT_ID/d' "$ENV_FILE"
+    sed -i '/BACKUP_CRON_SCHEDULE/d' "$ENV_FILE"
+    sed -i '/BACKUP_PROXY_ENABLED/d' "$ENV_FILE"
+    sed -i '/BACKUP_PROXY_URL/d' "$ENV_FILE"
+
+    local temp_cron=$(mktemp)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)